metaseq/sequence_generator.py

```diff
@@ -249,8 +250,9 @@ def _generate(
                 tokens[:, : step + 1],
                 incremental_state=incremental_states,
             )
-            model_out[0].div_(self.temperature)
-            lprobs = self.model.get_normalized_probs(model_out, log_probs=True)
+            model_predictions = model_out[0].float()
```

Comment:
Add a comment here for which bug this fixes? Don't want future eager optimizers to get excited about taking this out...

Reply:
we're currently in a live debugging session lmao